### PR TITLE
Revisit run tests

### PR DESCRIPTION
**Pull request by muryanto1**
migrate run_tests.py to use testsrunner and cdat_info -- replaced '-c uvcdat' with '-c cdat' in .travis.yml

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -20,8 +20,8 @@ aliases:
        conda config --set always_yes yes --set changeps1 no
        conda update -y -q conda
        conda config --set anaconda_upload no
-       conda create -q -n py2 -c cdat/label/nightly -c conda-forge -c cdat cdms2 nose flake8 mesalib image-compare matplotlib numpy=1.13 vcs "proj4<5" "python<3"
-       conda create -q -n py3 -c cdat/label/nightly -c conda-forge -c cdat cdms2 nose flake8 mesalib image-compare matplotlib numpy=1.13 vcs "proj4<5" "python>3"
+       conda create -q -n py2 -c cdat/label/nightly -c conda-forge -c cdat cdms2 cdat_info testsrunner  mesalib image-compare matplotlib numpy=1.13 vcs "proj4<5" "python<3"
+       conda create -q -n py3 -c cdat/label/nightly -c conda-forge -c cdat cdms2 cdat_info testsrunner  mesalib image-compare matplotlib numpy=1.13 vcs "proj4<5" "python>3"
```
Comment (doutriaux1) [inline .circleci/config.yml]: since you ported to cdat_info.TestRunner, we do not need image-compare anymore

File: .travis.yml [hunk 2]
```diff
@@ -13,8 +13,8 @@ before_install:
 script:
 - export UVCDAT_ANONYMOUS_LOG=False
 - export UVCDAT_SETUP_PATH=${HOME}
-- conda create -q -n py2 -c nesii/label/dev-esmf -c cdat/label/nightly -c conda-forge -c uvcdat cdms2 nose flake8 matplotlib mesalib image-compare vcs "proj4<5" "python<3"
-- conda create -q -n py3 -c nesii/label/dev-esmf -c cdat/label/nightly -c conda-forge -c uvcdat cdms2 nose flake8 matplotlib mesalib image-compare vcs "proj4<5" "python>3"
+- conda create -q -n py2 -c cdat/label/nightly -c conda-forge -c cdat cdms2 cdat_info testsrunner matplotlib mesalib image-compare vcs "proj4<5" "python<3"
+- conda create -q -n py3 -c cdat/label/nightly -c conda-forge -c cdat cdms2 cdat_info testsrunner  matplotlib mesalib image-compare vcs "proj4<5" "python>3"
```
Comment (doutriaux1) [inline .travis.yml]: same here `image-compare` is no longer needed

**Review by doutriaux1 — CHANGES_REQUESTED** [review]
@muryanto1 just a tiny tweak. thanks for putting this PR together!

**Comment by muryanto1** [comment]
Charles, I have incorporated your feedback (take out image-compare from the conda env create), and tests are passing.

**Review by doutriaux1 — APPROVED** [review]

**Pull request merged**